public/app/features/plugins/extensions/usePluginExtensions.test.tsx

```diff
@@ -157,41 +147,39 @@ describe('usePluginExtensions()', () => {
   });
 
   it('should only render the hook once', () => {
-    const spy = jest.spyOn(reactiveRegistry, 'asObservable');
-    const extensionPointId = 'plugins/foo/bar';
-    const usePluginExtensions = createUsePluginExtensions(reactiveRegistry, addedComponentsRegistry);
+    const addedComponentsRegistrySpy = jest.spyOn(registries.addedComponentsRegistry, 'asObservable');
+    const addedLinksRegistrySpy = jest.spyOn(registries.addedLinksRegistry, 'asObservable');
+    const extensionPointId = 'plugins/foo/bar/v1';
+    const usePluginExtensions = createUsePluginExtensions(registries);
 
     renderHook(() => usePluginExtensions({ extensionPointId }));
-    expect(spy).toHaveBeenCalledTimes(1);
+    expect(addedComponentsRegistrySpy).toHaveBeenCalledTimes(1);
+    expect(addedLinksRegistrySpy).toHaveBeenCalledTimes(1);
   });
 
-  it('should return the same extensions object if the context object is the same', () => {
-    const extensionPointId = 'plugins/foo/bar';
+  it.skip('should return the same extensions object if the context object is the same', async () => {
```

Comment:
I deliberately removed the hook cache and added [memoization](https://github.com/grafana/grafana/blob/f0115d43b5fa57daa09a2ae10c4d39f2de68e3d8/public/app/features/plugins/extensions/usePluginExtensions.tsx#L18). However, I doesn't seem to return the computed values the way I thought so this test no longer passes. Any ideas why this is not working? 

Adding back the cache would work of course, but it adds some complexity as this hook now uses two registries.

Reply:
Me and @leventebalogh made same change here https://github.com/grafana/grafana/pull/91648/files#diff-47339e5d1676beb0301fbc3fd96d6d6d94af817f795536c61acc09552b8cbf32L22 and I think that was passing tests.

Reply:
Ah okay! Appears to be the same [issue](https://drone.grafana.net/grafana/grafana/192335/2/6) in your PR 🤔